pkg/networkservice/common/onidle/server.go

```diff
@@ -60,6 +62,10 @@ func NewServer(ctx context.Context, notify func(), options ...Option) networkser
 		opt(t)
 	}
 
+	if t.timeout == 0 {
```

Comment:
We should probably *default* t.timeout = 0 above one line 56.

Reply:
I don't think default value 0 in the `onidle` server itself makes sense, because then we get the `null` server. So we would have to specify `onidle.WithTimeout()` always, which doesn't make sense for an optional parameter.
So, I went ahead and made `timeout` parameter mandatory.